README.md

```diff
@@ -31,5 +40,30 @@ too; replace `-full` with `-oss` in any of the above commands to install the
 OSS distribution. Note that the default distribution and OSS distribution of
 a product can not be installed at the same time.
 
+## How do I ensure my configuration is up to date?
```

Comment:
The instructions in this section are only necessary for this branch move, a `brew update && brew upgrade` normally updates homebrew things which I don't think we need to document. Consider moving this section to the new `Troubleshooting` section.

Reply:
I think it's also worth specifying it in the section title, maybe something like:
```suggestion
## How do I ensure my configuration is still up to date after the branch renaming?
```